spec/unit/chitter_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'chitter'
+require 'user'
+require 'database_helpers'
+
+
```

Comment:
Tests are well written and do a good job of testing the behaviour of the class

Reply:
Thank you